src/java/boa/aggregators/MeanAggregator.java

```diff
@@ -28,9 +28,17 @@
 
 	public void count(final String metadata) {
 		if (metadata == null)
+		{
 			this.count++;
+		}			
 		else
-			this.count += Long.parseLong(metadata);
+		{
+			try {
+				this.count += Long.parseLong(metadata);
+			} catch (NumberFormatException e) {
+				this.count += (long)Double.parseDouble(metadata);
```

Comment:
Ah yes you are write, this is count not sum.

So I guess now my question is why is the metadata sending a '1.0' as count.